docs/sources/configuration/_index.md

```diff
@@ -851,28 +851,31 @@ The `swift_storage_config` configures Swift as a general storage for different d
 [container_name: <string> | default = "cortex"]
 ```
 
-## hedging_config
+## hedging
 
-The `hedging_config` configures how to hedge requests for the storage.
+The `hedging` block configures how to hedge storage requests.
 
-Hedged requests is sending a secondary request until the first request has been outstanding for more than a configure expected latency
-for this class of requests.
-You should configure the latency based on your p99 of object store requests.
+The hedging implementation sends a second storage request once a first request has
+been outstanding for more than a configured expected latency for this class of requests.
+Calculate your latency to be when 99 percent of object storage requests have 
+seen responses.
```

Comment:
I'm wondering if we should provide an example query here.

```promql
histogram_quantile(0.99, 
  sum(rate(cortex_gcs_request_duration_seconds_bucket[5m])
) by (le))
```

I think we should stick with referring to percentiles here as this is a commonly-used term when discussing latencies.

```suggestion
Calculate your latency to be the 99th percentile of object storage response times.
```

Reply:
I've revised the description of the percentile.  Let's push off the task of adding an example for later, as I'm unclear on the future and use of the Cortex metric.